mod_wasm/build.sh

```diff
@@ -32,7 +39,6 @@ then
     HTTPD_DIR=$(realpath ../httpd)
     echo "\$HTTPD_DIR not defined! Using by default: $HTTPD_DIR"
 fi
-ARCH=$(uname -m)
```

Comment:
@ereslibre, will this work for both linux/x86_64 and linux/arm64?

Reply:
Yes, in those cases we can choose to export CC